charts/operator-wandb/charts/filestream/templates/_helpers.tpl

```diff
@@ -131,13 +131,17 @@ Create the name of the service account to use
 {{- define "filestream.fileStreamWorkerSource" -}}
 {{- if .Values.global.pubSub.enabled -}}
 pubsub:/{{ .Values.global.pubSub.project }}/{{ .Values.global.pubSub.filestreamTopic }}/{{ .Values.pubSub.subscription }}
-{{- else }}
-{{- end }}
-{{- end }}
+{{- else -}}
+{{- end -}}
+{{- end -}}
 
 {{- define "filestream.fileStreamWorkerStore" -}}
-{{- if .Values.global.bigTable.enabled -}}
-bigtablev3://{{ .Values.global.bigTable.project }}/{{ .Values.global.bigTable.instance }},bigtablev2://{{ .Values.global.bigTable.project }}/{{ .Values.global.bigTable.instance }}
-{{- else }}
-{{- end }}
-{{- end }}
+{{- $fileStreamStore := "" -}}
+{{- if .Values.global.bigtable.v3.enabled -}}
```

Comment:
Is this meant to be embedded?

Reply:
Should it be possible to enable v2 without v3.

Reply:
Yes, it is possible to enable V2 without V3.

Reply:
Okay, looks like this needs to be changed then.

Reply:
Let me fix this

Reply:
Fixed